notebook/automl_time_series_forecast.ipynb

```diff
@@ -1857,9 +5866,26 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 3,
+   "execution_count": 79,
    "metadata": {},
-   "outputs": [],
+   "outputs": [
+    {
+     "ename": "",
+     "evalue": "",
+     "output_type": "error",
+     "traceback": [
+      "\u001b[1;31mThe Kernel crashed while executing code in the the current cell or a previous cell. Please review the code in the cell(s) to identify a possible cause of the failure. Click <a href='https://aka.ms/vscodeJupyterKernelCrash'>here</a> for more info. View Jupyter <a href='command:jupyter.viewOutput'>log</a> for further details."
+     ]
+    },
+    {
+     "ename": "",
+     "evalue": "",
+     "output_type": "error",
+     "traceback": [
+      "\u001b[1;31mCanceled future for execute_request message before replies were done"
+     ]
+    }
+   ],
```

Comment:
This cell failed

Reply:
I'll look into this tomorrow. It's seems that the test will only fail with a certain built of MacOS?

Reply:
@jingdong00 Thanks! It fails many times when I use macOS. Can you save the output of every cell before pushing?

Reply:
@skzhang1  I've added all the cell output:). However, I do find the notebook to fail under macOS (but work with Linux)